test/unit/image-optimizer/match-remote-pattern.test.ts

```diff
@@ -0,0 +1,214 @@
+/* eslint-env jest */
+import {
+  matchRemotePattern as m,
+  hasMatch,
+} from 'next/dist/shared/lib/match-remote-pattern.js'
```

Comment:
```suggestion
} from 'next/dist/shared/lib/match-remote-pattern'
```
Are the types meant to be avoided here or why explicitly have the `.js` extension?